### ISSUE TITLE
Possible to specify a custom title in the indexmenu?

### ISSUE THREAD

**Issue by flelp** [samuelet/indexmenu#90]
Hello,

I've been reading the documentation and examples back and forth trying to figure out if it's possible to specify a custom/"fake" title to be displayed in the indexmenu.

Scenario:

We need a page that is actually a skeleton (template) to be copied and pasted to a new page. Currently this page holds the title (h1) "Documentation for X".

Since we've enabed the "useheading" option the indexmenu (navbar on the left) displays the same title. If possible, I would like indexmenu to use the title "Documentation template" instead.

Basically, this would be a quirk for this particular page only. All other pages should continue to "useheading" ;).

Is this possible with indexmenu or any other plugins/dokuwiki settings? 

Thanks a lot for your efforts with this plugin!

Thomas
